lib/services/sitemaps/index.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const elastic = require('../elastic'),
+  helpers = require('../elastic-helpers'),
+  h = require('highland'),
+  handlebars = require('handlebars'),
```

Comment:
I'm not super opinionated with this comment, but there is a clay org version of handlebars

Reply:
🤷‍♂️  we don't need any of the clay-handlebars stuff